SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -202,13 +202,17 @@
 
   private int maxTasksPerOfferPerRequest = 0;
 
-  private double longRunningUsedCpuWeightForOffer = 0.30;
+  private double longRunningUsedCpuWeightForOffer = 0.25;
 
-  private double longRunningUsedMemWeightForOffer = 0.70;
+  private double longRunningUsedMemWeightForOffer = 0.65;
```

Comment:
This actually changes the relative weights of cpu vs memory from memory being 2.333 times the weight of cpu to 2.666 times the weight of cpu. Would we instead want .27 and .63 to keep the relative weights of those two the same?

Reply:
Followed up in-person, will keep as is for now. mem > cpu > disk is preserved here which is the most important piece